features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin_new/source/src/app/components/Throttling/Subscription/Delete.jsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DeleteForeverIcon from '@material-ui/icons/DeleteForever';
+import FormDialogBase from 'AppComponents/AdminPages/Addons/FormDialogBase';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import API from 'AppData/api';
+
+/**
+ * Render delete dialog box.
+ * @param {JSON} props component props.
+ * @returns {JSX} Loading animation.
+ */
+function Delete(props) {
+    const restApi = new API();
+    const {
+        dataRow, subscriptionThrottlingPolicyList, updateList,
+    } = props;
+
+    const formSaveCallback = () => {
+        console.log(subscriptionThrottlingPolicyList);
```

Comment:
Shall we remove this console log ? 

BTW this should have caught by ESLint checks?
Have ignored this file(s) in `.eslintignore` file ?

Reply:
This file is not ignored in .eslintignore. But the build does not fail. I anyway removed this log.